client/signup/controller.js

```diff
@@ -99,12 +96,13 @@ export default {
 					stepName: stepName,
 					stepSectionName: stepSectionName
 				} )
-			),
-			document.getElementById( 'primary' )
 		);
+		context.secondary = null;
```

Comment:
Giving this some more thought, this is probably unneeded. This will cause [`unmountComponentAtNode`](https://facebook.github.io/react/docs/top-level-api.html#reactdom.unmountcomponentatnode) to run for `#secondary`. So if this piece of code wasn't calling `unmountComponentAtNode` before, we don't need to introduce it now -- continuing to assume it's not necessary. (Same for the other new occurrences of `context.secondary` in this diff.)